bears/vcs/git/GitCommitBear.py

```diff
@@ -39,7 +39,7 @@ def setup_dependencies(self):
             nltk.download([
                 'punkt',
                 'averaged_perceptron_tagger',
-            ])
+            ], quiet=True)
```

Comment:
This is preventing the messages to printed at all. They should be put out as log messages instead.

Reply:
Ah
Fair Point
I will attach the output to a logger :D

Reply:
I thought this issue is to prevent the printed messages.

Reply:
check the comments on the issue

Reply:
Updated and attached a logger